include/openthread/commissioner.h

```diff
@@ -243,6 +243,25 @@ OTAPI ThreadError OTCALL otCommissionerGeneratePSKc(otInstance *aInstance, const
                                                     uint8_t *aPSKc);
 
 /**
+ * This method sets Joiner timeout.
+ *
+ * @param[in]  aInstance  A pointer to an OpenThread instance.
+ * @param[in]  aTimeout   A time after which Joiners are automatically removed, in seconds.
+ *
+ */
+OTAPI void OTCALL otCommissionerSetJoinerTimeout(otInstance *aInstance, uint32_t aTimeout);
```

Comment:
What if we just made this timeout parameters a argument of the `otCommissionerAddJoiner` API? Then you wouldn't have to save the `uint32_t mJoinerTimeout` in the commissioner, saving those bytes.